### ISSUE TITLE
Selected Platforms/ Creat Workbook button/ System Mono combobox of Create a new Workbook page are not having 4.5:1 color contrast ratio.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#272]
### Steps to Reproduce

1. Launch Xamrin Workbook application.
2. In the "Create a new workbook" window that opens, observe the Colour Contrast ratio for Selected Platforms / System Mono combo box on selection and Create workbook button.
### Expected Behavior
Colour contrast ratio should be minimum 4.5:1

### Actual Behavior
Colour contrast ratio is lesser than 4.5:1 (i.e., 2.9:1)
as shown in the attached screen-shots.

### User Impact:
Low vision user will feel difficulty in seeing the text.

### Environment
MAC: macOS High Sierra [10.13.4]
Workbook - 1.4 (master/b3f92f9) 

### Log File

![issue 1](https://user-images.githubusercontent.com/22956130/39860737-f64c1316-542d-11e8-8f45-1936daa47d73.png)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to this bug:
A11yMAS
Accessibility
MAS1.4.3
Xamarin-Workbook-MAC
Wipro

**Comment by abock** [comment]
This is a stock macOS control. We do not influence or control the rendering of this control. The system-wide "Graphite" appearance may exhibit 4.5:1 contrast. Regardless, this is not something we can or should address in our code as it comes from macOS itself.

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1827
